src/components/AudioIndicator/AudioIndicator.tsx

```diff
@@ -33,15 +33,17 @@ export class AudioIndicator extends Component<Props, State> {
     }
 
     componentDidMount() {
-        if (!this.props.audioOn) {
+        const { audioOn, stream } = this.props;
+        const { foundAudioSignal } = this.state;
+        if (!audioOn || !stream) {
```

Comment:
```suggestion
        const { audioOn, stream } = this.props;
        
        if (!audioOn || !stream) {
          return;
        }
        
        const { foundAudioSignal } = this.state;
```
if the condition is met. so, the `foundAudioSignal` has been fetched for no reason.
I suggest changing the order of code here as I've suggested.

Reply:
done